types/time.go

```diff
@@ -1270,9 +1549,15 @@ func parseTime(sc *stmtctx.StatementContext, str string, tp byte, fsp int, isFlo
 	return t, nil
 }
 
+func parseTime(sc *stmtctx.StatementContext, str string, tp byte, fsp int, isFloat bool) (Time, error) {
```

Comment:
There is only one place use this function now? And this function can be just one line?
If so, we can remove this function and use `parseTimeDecideFsp` directly?